rotkehlchen/chain/ethereum/uniswap/uniswap.py

```diff
@@ -392,8 +392,8 @@ def _get_trades(
                 location=Location.UNISWAP,
                 address=address,
             )
-            db_trades = self.swaps_to_trades(db_swaps)
-            if db_trades:
+            if db_swaps:
+                db_trades = self.swaps_to_trades(db_swaps)
```

Comment:
@vnavascues Thanks for finding the bug! Can you though solve it by adding the check inside `swaps_to_trades()`?

Also perhaps unit test that function with an empty list?

Reply:
I will. I didn't want to mess up this function, but better have it there as well.

Reply:
The function is messed up already :smile: 

It takes a list and returns a list, but if the list it takes has length 0 it raises an exception. The bug that you found is in the function :wink: